benchmarks/renderToString.js

```diff
@@ -4,31 +4,59 @@
  * compare renderToString
  */
 
-const cp = require('child_process');
+const Benchmark = require('benchmark');
 
-const limit = 10;
+const Rax = require('rax');
+const raxRenderToString = require('rax-server-renderer').renderToString;
+const React = require('react');
+const ReactDOMServer = require('react-dom/server');
```

Comment:
@aickin I don't know how to `require` ReactDomServer from `dist`, I try this `const ReactDOMServer = require('react-dom/dist/react-dom-server.min');`, but get the error:

```
➜  server-side-rendering-comparison git:(use-benchmark.js) ✗ NODE_ENV=production node benchmarks/renderToString.js
/Users/sobear/work/ssr/server-side-rendering-comparison/node_modules/.15.4.2@react-dom/dist/react-dom.min.js:15
o(n,r);case"topSelectionChange":if(f)break;case"topKeyDown":case"topKeyUp":return o(n,r)}return null},didPutListener:function(e,t,n){"onSelect"===t&&(_=!0)}};t.exports=C},{111:111,123:123,132:132,141:141,19:19,33:33,56:56,80:80}],75:[function(e,t,n){"use strict";function r(e){return"."+e._rootNodeID}function o(e){return"button"===e||"input"===e||"select"===e||"textarea"===e}var i=e(113),a=e(122),s=e(19),u=e(33),l=e(76),c=e(77),p=e(80),d=e(81),f=e(83),h=e(84),m=e(79),v=e(85),g=e(86),y=e(87),_=e(88),C=e(129),b=e(99),E=(e(137),{}),x={};["abort","animationEnd","animationIteration","animationStart","blur","canPlay","canPlayThrough","click","contextMenu","copy","cut","doubleClick","drag","dragEnd","dragEnter","dragExit","dragLeave","dragOver","dragStart","drop","durationChange","emptied","encrypted","ended","error","focus","input","invalid","keyDown","keyPress","keyUp","load","loadedData","loadedMetadat

TypeError: Cannot read property 'ReactCurrentOwner' of undefined
    at Object.r.120 (/Users/sobear/work/ssr/server-side-rendering-comparison/node_modules/.15.4.2@react-dom/dist/react-dom.min.js:15:20480)
    at o (/Users/sobear/work/ssr/server-side-rendering-comparison/node_modules/.15.4.2@react-dom/dist/react-dom.min.js:12:624)
    at /Users/sobear/work/ssr/server-side-rendering-comparison/node_modules/.15.4.2@react-dom/dist/react-dom.min.js:12:675
    at Object.r.61.113 (/Users/sobear/work/ssr/server-side-rendering-comparison/node_modules/.15.4.2@react-dom/dist/react-dom.min.js:14:14876)
    at o (/Users/sobear/work/ssr/server-side-rendering-comparison/node_modules/.15.4.2@react-dom/dist/react-dom.min.js:12:624)
    at /Users/sobear/work/ssr/server-side-rendering-comparison/node_modules/.15.4.2@react-dom/dist/react-dom.min.js:12:675
    at Object.r.31.10 (/Users/sobear/work/ssr/server-side-rendering-comparison/node_modules/.15.4.2@react-dom/dist/react-dom.min.js:13:14983)
    at o (/Users/sobear/work/ssr/server-side-rendering-comparison/node_modules/.15.4.2@react-dom/dist/react-dom.min.js:12:624)
    at /Users/sobear/work/ssr/server-side-rendering-comparison/node_modules/.15.4.2@react-dom/dist/react-dom.min.js:12:675
    at Object.r.47.1 (/Users/sobear/work/ssr/server-side-rendering-comparison/node_modules/.15.4.2@react-dom/dist/react-dom.min.js:14:4823)
```

Maybe you can help me, thank you.